### PR TITLE
v0.1.1

### PR DESCRIPTION
**Pull request by pellared**
Review: https://github.com/golang-templates/library/blob/8495f35f9d6adb09c16e9709307bee7a77160b00/README.md

README refined.
New FAQ section:
- Why Mage instead of Make
- Why nothing for GoLand
- Why GitHub Actions, not any other CI server

CircleCI config works: https://app.circleci.com/pipelines/github/pellared/test-circleci/1/workflows/8236f2e3-3f39-443a-80b7-994b93b3da5b/jobs/1

Makefile also tested

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
...
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+```
+
+1. Remove Mage installation from [Dockerfile](Dockerfile).
+1. Update [docker-compose.yml](docker-compose.yml) and [.vscode/tasks.json](.vscode/tasks.json) to use `make`.
+
+If you want to use Make on bare-metal Windows, then you can:
+
+1. use [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Visual Studio Code Remote SSH extension](https://code.visualstudio.com/docs/remote/wsl),
```
Comment (agabrys) [inline README.md:98]:
```suggestion
1. Use [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Visual Studio Code Remote SSH extension](https://code.visualstudio.com/docs/remote/wsl),
```

Half of the points start with lowercase and the second half with uppercase.

File: README.md [hunk 2]
```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
...
+1. install [Make Windows port to Git Bash](https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058).
```
Comment (agabrys) [inline README.md:99]:
```suggestion
1. Install [Make Windows port to Git Bash](https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058).
```

File: README.md [hunk 3]
```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
...
+
+### Why nothing for GoLand
+
+The maintainer does not use GoLand. Fell free to create a pull request which should contain:
+
+- GoLand config to use golangci-lint,
+- GoLand config for Mage targets (like [.vscode/tasks.json](.vscode/tasks.json) for Visua Studio Code),
+- update [.gitignore](.gitignore) if required.
+- update README(README.md) if possible.
```
Comment (agabrys) [inline README.md:108]:
```suggestion
- update [README](README.md) if possible.
```

File: README.md [hunk 4]
```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
...
+
+1. use [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Visual Studio Code Remote SSH extension](https://code.visualstudio.com/docs/remote/wsl),
+1. install [Make Windows port to Git Bash](https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058).
+
+### Why nothing for GoLand
+
+The maintainer does not use GoLand. Fell free to create a pull request which should contain:
+
+- GoLand config to use golangci-lint,
+- GoLand config for Mage targets (like [.vscode/tasks.json](.vscode/tasks.json) for Visua Studio Code),
```
Comment (agabrys) [inline README.md:106]:
```suggestion
- GoLand configuration for Mage targets (like [.vscode/tasks.json](.vscode/tasks.json) for Visual Studio Code),
```

File: README.md [hunk 5]
```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
...
+However, changing to any other CI server should be very simple, becasue this repo uses Docker Compose to run CI build to make the transition easy.
+
+For [CircleCI](https://circleci.com/docs/2.0/executor-types/#using-machine) create `.circleci/config.yml` file:
+
+```yml
+version: 2.1
+jobs:
+  build:
+    machine:
+      image: ubuntu-1604:201903-01
```
Comment (agabrys) [inline README.md:122]: Is it impossible to use `latest`?

File: README.md [hunk 6]
```diff
@@ -46,7 +45,85 @@ Take notice that this project is build in a way that gives the developer a lot o
...
+If you want to use Make on bare-metal Windows, then you can:
+
+1. use [WSL (Windows Subsystem for Linux)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and [Visual Studio Code Remote SSH extension](https://code.visualstudio.com/docs/remote/wsl),
+1. install [Make Windows port to Git Bash](https://gist.github.com/evanwill/0207876c3243bbb6863e65ec5dc3f058).
+
+### Why nothing for GoLand
+
+The maintainer does not use GoLand. Fell free to create a pull request which should contain:
+
+- GoLand config to use golangci-lint,
```
Comment (agabrys) [inline README.md:105]:
```suggestion
- GoLand configuration to use GolangCI-Lint,
```

File: README.md (hunk 5 above)
Comment (pellared) [reply README.md:122]: Does not work as you would think
https://discuss.circleci.com/t/default-machine-executor-image-update/29308
https://discuss.circleci.com/t/new-docker-versions-for-machine-images-on-circleci-2-0-on-2017-11-29/18144

File: README.md (hunk 2 above)
Comment (pellared) [reply README.md:99]: there is a comma before.

File: README.md (hunk 1 above)
Comment (pellared) [reply README.md]: Changed. Please review

File: README.md (hunk 2 above)
Comment (pellared) [reply README.md]: Changed. Please review

File: README.md [hunk 7]
```diff
@@ -29,7 +28,7 @@ It includes:
 
 ### Setup Development Environment
 
-Take notice that this project is build in a way that gives the developer a lot of freedom on the development environment setup. Below you can find proposals when using Visual Studio Code. 
+Take notice that this project is build in a way that gives the developer a lot of freedom on the development environment setup. Below you can find proposals when using Visual Studio Code.
```
[the remark this answers is not in the document]
Comment (pellared) [reply README.md:31]: Refined please review

**Pull request merged**